packages/@react-aria/i18n/src/context.tsx

```diff
@@ -26,7 +26,7 @@ const I18nContext = React.createContext<Locale | null>(null);
 /**
  * Provides the locale for the application to all child components.
  */
-export function I18nProvider(props: I18nProviderProps): ReactNode {
+export function I18nProvider(props: I18nProviderProps): JSX.Element {
```

Comment:
but this one was https://app.unpkg.com/@react-aria/i18n@3.12.7/files/dist/types.d.ts